Plugins/BTCPayServer.RockstarDev.Plugins.Payroll/Views/PayrollUser/List.cshtml

```diff
@@ -16,7 +16,7 @@
     <div class="sticky-header d-sm-flex align-items-center justify-content-between">
         <h2 class="mb-0">@ViewData["Title"]</h2>
         <div class="d-flex gap-3 mt-3 mt-sm-0">
-            <a asp-controller="PayrollUser" asp-action="Create" asp-route-storeId="@storeId" class="btn btn-primary">Create User</a>
+			<a asp-controller="PayrollUser" asp-action="Create" asp-route-storeId="@storeId" class="btn btn-primary">Create User</a>
```

Comment:
Not sure why extra formatting here. @TChukwuleta if you're using Rider you can do CTRL+ENTER to format file. Or run full format on your end as well on Payroll project - see if .editorconfig works on your machine.